woocommerce/woocommerce
# wp-cli make-json and Loco Translate support for admin dashboard.

### All Submissions:






### Changes proposed in this Pull Request:



Closes #35500  .






### How to test the changes in this Pull Request:



1. Set WordPress locale to a language that doesn't have official translations for WooCommerce (Ex. Azerbaijan).
2. Create translations for that language in Loco Translate. There are 3 possible locations for PO file, you can choose any, It's irrelevant.
*languages/plugins/woocommerce-az.po
*languages/loco/plugins/woocommerce-az.po
*plugins/woocommerce/i18n/languages/woocommerce-az.po
3. Translate couple random words and save. This will compile MO file and generate JSON files in the chosen directory with following format "woocommerce-{locale}-{MD5}.json". AKA "chunks".
4. De-Activate and Activate WooCommerce to trigger the associated action.
5. In the chosen directory, chunk files should be combined into single woocommerce-{locale}-wc-admin-app.json file.


### Short summary of bug
Due to file format difference between the official and generated chunk files, WooCommerce was unable to generate combined JSON file.

### Other information:




### FOR PR REVIEWER ONLY:
